initializr-generator/src/main/java/io/spring/initializr/generator/buildsystem/maven/MavenBuildWriter.java

```diff
@@ -57,6 +64,7 @@
 	 */
 	public void writeTo(IndentingWriter writer, MavenBuild build) {
 		MavenBuildSettings settings = build.getSettings();
+		MavenDistributionManagement distributionManagement = build.getDistributionManagement();
```

Comment:
I don't think we need to initialize this here right? We can do that in the only method that uses it.